glide-core/src/client/value_conversion.rs

```diff
@@ -411,11 +442,24 @@ fn convert_array_elements(
     Ok(Value::Array(converted_array))
 }
 
-fn convert_array_to_map(
+/// Converts a 2-dimensional array to map, where the first element is the map key, and the second element
```

Comment:
Is the input supposed to be something like [[key1, value1], [key2, value2]]?
```suggestion
/// Converts a 2-dimensional array to a map, where the first element in each inner array is the map key, and the second element
```